example/py-scr3.py

```diff
@@ -18,17 +18,17 @@
 params["Ham_im_prefix"] = rt+"0_H_dia_"
```

Comment:
In the case of nac_method = 0 or 2, these should be just "0_Ham_" , no mention of anything "diabatic" - everything is adiabatic (non-relativistic), same for nac_method = 1 - everything is adiabatic (relativistic). We can mention "diabatic" only in case nac_method=3